doc/Makefile

```diff
@@ -47,13 +47,13 @@ api:
 
 
 linkcheck:
-	$(SPHINXBUILD) -b linkcheck $(ALLSPHINXOPTS) $(BUILDDIR)/linkcheck
+	$(SPHINXBUILD) -b linkcheck -j auto $(ALLSPHINXOPTS) $(BUILDDIR)/linkcheck
 	@echo
 	@echo "Link check complete; look for any errors in the above output " \
 	      "or in $(BUILDDIR)/linkcheck/output.txt."
 
 doctest:
```

Comment:
Not sure if we really need the `doctest` target here. Will we add any doctests in ReST files?

Reply:
Never used `make doctest` to be honest. The `git blame` on this goes right back to 2017 at c272bd350fdae4ac3ca1025ecd2f4faf99a0d907.

Reply:
I think we can just remove it. We can still add it back if we decide to write more ReST files in the `doc` directory.

Reply:
Ok, done at 89f1f941a37bd2ccfe470f56265f273e9c93721f.